src/components/input/input-checkbox-radio-styles.jsx

```diff
@@ -0,0 +1,137 @@
+import { createStyleSheet } from 'jss-theme-reactor';
+import variables from '../../utilities/variables';
```

Comment:
I added it because variables.easeOut is used on multiple places. Lets talk about how we want to solve it.